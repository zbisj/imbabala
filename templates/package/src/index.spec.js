import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Example from ".";

describe("Example", () => {
  test("renders correctly", () => {
    const tree = renderer.create(<Example />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
