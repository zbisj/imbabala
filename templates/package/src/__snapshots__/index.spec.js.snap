// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example renders correctly 1`] = `<div />`;
